lib/seattleflu/id3c/cli/command/etl/redcap_det_scan.py

```diff
@@ -889,7 +882,13 @@ def cast_to_string(string: str) -> Optional[str]:
 
     def cast_to_integer(string: str) -> Optional[int]:
         try:
-            return int(string)
+            # Age Ceiling
+            if question_id == 'age':
+                return age_ceiling(int(string))
+            elif question_id == 'age_months':
+                return age_ceiling(int(string) / 12) * 12
+            else:
+                return int(string)
```

Comment:
Do you mean the name of the function `cast_to_integer` is misleading because `age_ceiling` returns a float?

Reply:
The type hint in `age_ceiling` indicates it returns a float, but that only happens if the passed `age` argument is a float. Since the age values we pass are integers, these will return int as expected.

The issue of reporting age >= 1y as a float is actually  within the FHIR ETL not here.

Reply:
> Do you mean the name of the function `cast_to_integer` is misleading because `age_ceiling` returns a float?

Yes.

> The type hint in `age_ceiling` indicates it returns a float, but that only happens if the passed age argument is a float. Since the age values we pass are integers, these will return int as expected.

> The issue of reporting age >= 1y as a float is actually within the FHIR ETL not here.

Ah, then perhaps all we need to do is update the type hint for the return value of `age_ceiling`.

Reply:
I restructured the function so that everything gets returned as an int in the latest push.